modules/ts/include/opencv2/ts/ts_perf.hpp

```diff
@@ -169,7 +169,8 @@ CV_ENUM(MatDepth, CV_8U, CV_8S, CV_16U, CV_16S, CV_32S, CV_32F, CV_64F, CV_USRTY
 enum ERROR_TYPE
 {
     ERROR_ABSOLUTE = 0,
-    ERROR_RELATIVE = 1
+    ERROR_RELATIVE = 1,
+    ERROR_LOOSE = 2
```

Comment:
Please revert these changes in test framework.
You can use `SANITY_CHECK_NOTHING()` in perf tests if reliable check is not possible (like these warp operations).

Reply:
I reverted ```ERROR_LOOSE``` and added ```SANITY_CHECK_NOTHING``` in some tests
Thank you